From 79404e05d7397f40411ceb5c67970c1d59ce6299 Mon Sep 17 00:00:00 2001
Message-Id: <patch-1.1-79404e05d73-20220407T185645Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Tue, 5 Apr 2022 21:48:43 +0200
Subject: [PATCH] doc txt & -h consistency: fix recent "cat-file" inconsistency
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The cat-file SYNOPSIS and help output were made consistent in
83dc443439c (cat-file: don't whitespace-pad "(...)" in SYNOPSIS and
usage output, 2022-01-10).

They then drifted apart again in 440c705ea63 (cat-file: add
--batch-command mode, 2022-02-18). Let's fix that and list the new
--batch-command option in the SYNOPSIS section.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Documentation/git-cat-file.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/git-cat-file.txt b/Documentation/git-cat-file.txt
index 70c5b4f12d1..24a811f0ef6 100644
--- a/Documentation/git-cat-file.txt
+++ b/Documentation/git-cat-file.txt
@@ -12,7 +12,7 @@ SYNOPSIS
 'git cat-file' <type> <object>
 'git cat-file' (-e | -p) <object>
 'git cat-file' (-t | -s) [--allow-unknown-type] <object>
-'git cat-file' (--batch | --batch-check) [--batch-all-objects]
+'git cat-file' (--batch | --batch-check | --batch-command) [--batch-all-objects]
 	     [--buffer] [--follow-symlinks] [--unordered]
 	     [--textconv | --filters]
 'git cat-file' (--textconv | --filters)
-- 
2.36.0.rc0.849.g2d5b5d9ab01

